code/LarryWright/make_change.py

```diff
@@ -3,17 +3,31 @@
 dime = .10
 penny = 0.01
 
-penny_results = total_amount // penny
-dime_results = total_amount // dime
 quarters_results = total_amount // quarter
-
-
 print(quarters_results)
 
 quarter_change = quarters_results * quarter
+print(quarter_change)
+
+remainder_1 = total_amount - quarter_change
+print(round(remainder_1, 2))
+
+
+dime_results = round(remainder_1, 2) // dime
+print(dime_results)
+
+dime_change = dime_results * dime
+print(dime_change)
+
+remainder_2 = round(remainder_1, 2) - dime_change
+print(round(remainder_2, 2))
 
-remainder_1 = total_amount - quarter_change 
-print(remainder_1)
 
+penny_results = round(remainder_2, 2) // penny
+print(penny_results)
 
+penny_change = penny_results * penny
+print(penny_change)
 
+remainder_3 = round(remainder_2, 2) - penny_change

```

Comment:
The code produces a trail of print statements which aren't describing in a clear way the amount of quarters dimes and pennies.